heron/scheduler-core/src/java/com/twitter/heron/scheduler/RuntimeManagerMain.java

```diff
@@ -318,7 +319,9 @@ public static void main(String[] args)
       // SUPPRESS CHECKSTYLE IllegalCatch
     } catch (UpdateDryRunResponse response) {
       LOG.log(Level.FINE, "Sending out dry-run response");
-      System.out.print(runtimeManagerMain.renderDryRunResponse(response));
+      // Output may contain UTF-8 characters, so we should print using UTF-8 encoding
+      PrintStream out = new PrintStream(System.out, true, "UTF-8");
```

Comment:
Instead of using strings, can we use `StandardCharsets.UTF-8` for a `Locale` or `StandardCharsets.UTF-8.name()`for a string representation? This is consistency with how locales are specified in the rest of the codebase. 

Also please allow at least a few business hours of time for people to review non-critical patches before merging.